pkg/apis/servicecatalog/types.go

```diff
@@ -88,6 +112,15 @@ type ClusterServiceBrokerSpec struct {
 	AuthInfo *ClusterServiceBrokerAuthInfo
 }
 
+// ServiceBrokerSpec represents a description of a Broker.
+type ServiceBrokerSpec struct {
```

Comment:
would it be possible to move `ServiceBrokerAuthInfo` into `CommonServiceBrokerSpec` ? or do they have different futures?

Reply:
They’re different and have different futures